### PR TITLE
fix(suite): fix link in pass modal

### LINKED ISSUE #15466: Passphrase flow is missing URL to the Knowledge Base
**Describe the bug**
Passphrase entry flow is missing URL to the Knowledge Base

**Info:**

-   Suite Version: 24.10

**How to reproduce**

1. Have a Trezor connected and unlocked, initialized device, so you can se the dashboard or some account.
2. Open wallet switcher in the upper left corner
3. Click the [+ passphrase wallet] button
4. Observe the first "It's important to first learn how a passphrase works" bullet point, it's missing a URL to the Knowledge Base

**Expected behavior**
The "It's important to first learn how a passphrase works" text should have a URL on the "learn how a passphrase works" part of the text (it should be underscored and have a hover effect like any other text URL)

It should link to: https://trezor.io/learn/a/passphrases-and-hidden-wallets

**Screenshots**
Image

**Additional context**

### PR DESCRIPTION
**Pull request by MiroslavProchazka**
Resolve https://github.com/trezor/trezor-suite/issues/15466
Before
image

After
image

### CODE REVIEW

File: packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseModal.tsx [hunk 1]
```diff
@@ -128,14 +127,15 @@ export const PassphraseModal = ({ device }: PassphraseModalProps) => {
...
-                                        a: chunks => (
-                                            <TrezorLink
-                                                target="_blank"
-                                                variant="underline"
+                                        learnLink: chunks => (
+                                            <a
                                                 href={HELP_CENTER_PASSPHRASE_URL}
+                                                target="_blank"
+                                                rel="noopener noreferrer"
+                                                style={{ textDecoration: 'underline' }}
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseModal.tsx:135]: we dont use it like this

File: packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseModal.tsx [hunk 2]
```diff
@@ -128,14 +127,15 @@ export const PassphraseModal = ({ device }: PassphraseModalProps) => {
                                 <Translation
                                     id="TR_PASSPHRASE_DESCRIPTION_ITEM1"
                                     values={{
-                                        a: chunks => (
```
Comment (tomasklim) [inline packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseModal.tsx]: Changes in this file are not needed. Just change text to `"It's important to first <a>learn how a passphrase works</a>."`

**Review by tomasklim — CHANGES_REQUESTED** [review]

**Review by tomasklim — APPROVED** [review]
LGTM

**Review by jvaclavik — APPROVED** [review]
Nice!

**Review by seibei-iguchi — APPROVED** [review]

**Pull request merged**